changelog.html

```diff
@@ -72,6 +72,9 @@
   <li class="rfe">
     Added missing translations for Polish language.
     (<a href="https://github.com/jenkinsci/jenkins/pull/1986">pull 1986</a>)
+  <li class="bug">
+    A custom update center is not consulted for Tool installer metadata.
```

Comment:
Would make sense to convert the message to "Retrieve tool installer metadata from all update sites" or whatever